doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1427,9 +1485,14 @@ An `http` object contains the following subfields:
 
 + `url` : URL referencing the service which is invoked when a notification is generated. A NGSIv2
   compliant server must support `http` URL schema, other schemas could also be supported.
+
+An `httpExtended` object conatins the following subfields. If `httpExtened` is used, then
+"Notification templates" will be used (see specific section).
+
 + `headers` (optional): a key-map of HTTP headers that are included in the notification messages.
 + `qs` (optional): a key-map of URL query parameters that are included in the notification message.
-+ `method` (optional): the method to use for sending the notification (default is POST)
++ `method` (optional): the method to use for sending the notification (default is POST). Only valid
+  HTTP methods are allowed, otherwise a 400 Bad Request error will be returned.
```

Comment:
otherwise ...

Only valid HTTP methods are allowed. On receiving an invalid HTTP method, a 400 Bad Request error is returned.